.github/workflows/python-publish.yml

```diff
@@ -22,7 +25,11 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install setuptools wheel twine
+    - name: Test
+      run: |
+        make test
     - name: Build and publish
+      if: contains(github.ref, 'master')
```

Comment:
I think we can remove this line, the release tag is already attached to the master branch.

Reply:
You are right. Thanks for comment.